app/views/shared/_user_menu.html.erb

```diff
@@ -0,0 +1,64 @@
+<div
+    class="w-full border-t border-gray-200/50 pt-6 pb-2 pr-2 bg-black flex flex-col gap-2"
+    data-controller="click-toggle"
+    data-click-toggle-flippable-class="hidden"
+    >
+
+    <a href="#"
+       data-action="click->click-toggle#toggleClasses:prevent"
+       class="flex justfiy-left gap-2 rounded-lg text-white text-sm hover:bg-[#28282b] p-2"
+    >
+      <%= icon "cog" , size: 20 %>
+      Settings
+    </a>
+
+    <a href="/logout" class="flex justify-left gap-2 hover:bg-[#28282b] rounded-lg text-white text-sm p-2">
+    <%= heroicon "arrow-right-start-on-rectangle" , options: { class: 'h-5 w-5 text-gray-300' } %>
+    <p>Logout</p>
+  </a>
+
+  <div data-click-toggle-target="destination"
+       class="fixed inset-0 z-50 hidden"
+       >
+    <div
+      class="absolute inset-0 bg-[rgba(0,0,0,0.5)] flex justify-center items-center"
+      data-action="click->click-toggle#toggleClasses:self"
+    >
+    <div class="bg-[#343541] w-[400px] text-gray-200 p-6 rounded shadow-lg flex flex-col gap-4">
+      <div class="flex justify-between items-baseline">
+        <h2 class="text-2xl font-bold">Settings</h2>
+        <button class="text-gray-300 hover:text-gray-400 text-3xl" data-action="click->click-toggle#toggleClasses">
+          &times;
+        </button>
+      </div>
+
+        <%= form_with(
+          model: Current.user,
+          method: :put,
+          url: user_path(current_user),
```

Comment:
`user_path(Current.user)`? I didn't know Ali had added `def current_user`, but I think we should remove that and lean into Current.user everywhere.

Reply:
Agreed, I probably just missed this one when I moved this code over.